teuthology/task/install/util.py

```diff
@@ -67,8 +67,9 @@ def ship_utilities(ctx, config):
     filenames = []
 
     log.info('Shipping valgrind.supp...')
+    assert 'suite_path' in ctx.config
     with file(
-        os.path.join(os.path.dirname(__file__), 'valgrind.supp'),
+        os.path.join(ctx.config['suite_path'], 'valgrind.supp'),
```

Comment:
I think this would cause problems with teuthology runs on older versions of Ceph?

Reply:
Indeed, that could happen. However, the whitelist file is now ported to older Ceph branches as well.

Reply:
I think the only think we might add here is a try/catch or whatever so that we do not fail hard if the valgrind.supp file isn't present in the ceph repo for some reason

Reply:
Added try/catch + log message to not fail silently.
There are first greens in http://pulpito.front.sepia.ceph.com/rzarzynski-2019-04-24_14:40:46-rados-wip-sage-testing-2019-04-23-1419-distro-basic-smithi/.